ui/tests/acceptance/client-history-test.js

```diff
@@ -0,0 +1,282 @@
+    const licenseStart = startOfMonth(subMonths(new Date(), 6));
+    const licenseEnd = addMonths(new Date(), 6);
+    const license = generateLicenseResponse(licenseStart, licenseEnd);
+    const config = generateConfigResponse({ enabled: 'default-disable' });
+    // const activity = generateActivityResponse(0, licenseStart, licenseEnd);
+    this.server = new Pretender(function () {
+      this.get('/v1/sys/license/status', () => sendResponse(license));
+      this.get('/v1/sys/internal/counters/activity', () => sendResponse(null, 204));
+      this.get('/v1/sys/internal/counters/config', () => sendResponse(config));
+      this.get('/v1/sys/version-history', () => sendResponse({ keys: [] }));
+      this.get('/v1/sys/health', this.passthrough);
+      this.get('/v1/sys/seal-status', this.passthrough);
+      this.post('/v1/sys/capabilities-self', this.passthrough);
+      this.get('/v1/sys/feature-flags', this.passthrough);
+    });
+    await visit('/vault/clients/history');
+    assert.equal(currentURL(), '/vault/clients/history');
+    assert.dom(SELECTORS.activeTab).hasText('History', 'history tab is active');
+
+    assert.dom('[data-test-tracking-disabled] .message-title').hasText('Tracking is disabled');
+    assert.dom(SELECTORS.emptyStateTitle).hasText('No data received');
+    assert.dom(SELECTORS.filterBar).exists('Shows filter bar to search previous dates');
+    assert.dom(SELECTORS.usageStats).doesNotExist('No usage stats');
+    // TODO: has no end filter date
+  });
+
+  test('shows warning when config off, no data, queries unavailable', async function (assert) {
+    const licenseStart = startOfMonth(subMonths(new Date(), 6));
+    const licenseEnd = addMonths(new Date(), 6);
+    const license = generateLicenseResponse(licenseStart, licenseEnd);
+    const config = generateConfigResponse({ enabled: 'default-disable', queries_available: false });
+    // const activity = generateActivityResponse(0, licenseStart, licenseEnd);
+    this.server = new Pretender(function () {
+      this.get('/v1/sys/license/status', () => sendResponse(license));
+      this.get('/v1/sys/internal/counters/activity', () => sendResponse(null, 204));
+      this.get('/v1/sys/internal/counters/config', () => sendResponse(config));
+      this.get('/v1/sys/version-history', () => sendResponse({ keys: [] }));
+      this.get('/v1/sys/health', this.passthrough);
+      this.get('/v1/sys/seal-status', this.passthrough);
+      this.post('/v1/sys/capabilities-self', this.passthrough);
+    });
+    await visit('/vault/clients/history');
+    assert.equal(currentURL(), '/vault/clients/history');
+    assert.dom(SELECTORS.activeTab).hasText('History', 'history tab is active');
+    assert.dom(SELECTORS.emptyStateTitle).hasText('Data tracking is disabled');
+    assert.dom(SELECTORS.filterBar).doesNotExist('Filter bar is hidden when no data available');
+    // TODO: Hide billing start month?
```

Comment:
I believe they should still be able to see/edit the start month to query historical data

Reply:
Pushed a commit updating EmptyState text per Ivana for no config off, but potential historical data